### ISSUE TITLE
Unable to add new card on the billing section

### ISSUE THREAD

**Issue by rohitbindra** [a85/POSTMan-Chrome-Extension#8230]
**Describe the bug**
Unable to add new card on the billing section, the error is "'StripeCheckout' is not defined". I have checked on chrome and edge browsers.

**To Reproduce
![error]
**
1. Go to 'Billing'
2. Click on 'Payment Methods'
3. Click on 'Add a new Card'
4. See error 'StripeCheckout' is not defined"

**Expected behavior**
It should allow to add a new card as the existing has expired.

**Screenshots**
(https://user-images.githubusercontent.com/39771471/77347396-11a76280-6d38-11ea-9abe-99487e3d723c.png)

**App information (please complete the following information):**
 - Chrome Version 80.0.3987.149
 - Postman Version NA
 - OS: [Win 10]

**Additional context**

**Comment by smitit173** [comment]
Hi Rohit, thanks for your feedback. Could you please send the devtools screenshot as well so that we can confirm it's not a timing issue.

**Comment by rohitbindra** [comment]
Hi, I'm not using any tool just doing the administration, only have MS office on my laptop.

Regards,
Rohit

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by smitit173** [comment]
Hey, there is no need for any tool as such. Since the error has been experienced in the chrome browser you can follow these steps :
1.  press `ctrl + shift + J ` this will open chrome devtools for you
2. You will see a **Network** header there 
3. Under the Network header there will be a **JS** section
4. Reload the page using `ctrl + shift + R` you will see a list of files under the JS section
5. You will find a file called 'checkout.js'. and there will be time and status associated to it.
 
Could you please let me know what is the status and the time associated with checkout.js. 
It indicates the time that it takes to load the script. That is what might be causing your error.

Attaching an image for your reference : 
<img width="1273" alt="stripe checkout not defined error" src="https://user-images.githubusercontent.com/26775288/77413802-f2ffa500-6de5-11ea-9f32-ef43c6439e16.png">

Regards,
Smit

**Comment by rohitbindra** [comment]
![image](https://user-images.githubusercontent.com/39771471/77440247-60143a00-6de8-11ea-9013-f21fe9be5762.png)

**Comment by smitit173** [comment]
Hi Rohit ,
 We can see that the stripe checkout library isn't getting loaded. That is the reason you are getting the error. There are several approaches / steps that you can take : 
1. Clear the cache/ browsing data for this site. To learn how to clear cache for the browser you can see the image for reference.
<img width="742" alt="Screen Shot 2020-03-24 at 10 48 31 PM" src="https://user-images.githubusercontent.com/26775288/77456759-b5b90880-6e21-11ea-82fe-1b5b1564c8d0.png">

2. If this does not solve the error you can check whether stripe.com is whitelisted. If you have an ad blocker enabled please ensure that they are not blocking stripe.com. Furthermore, If you are using chrome enterprise version which supports whitelisting and blacklisting URLs, you can have a look to ensure that stripe.com is enabled

This should probably solve the problem for you. 

Cheers, 
Smit 🎉

**Comment by rohitbindra** [comment]
Thanks for looking into it. It failed from the android mobile browser as well but worked on a different laptop, maybe something was getting blocked on my company laptop due to some company policy.

You should definitely get the scenario tested on mobile browser, everything shows green but no output..
